torch/csrc/dynamo/cpython_defs.h

```diff
@@ -1,6 +1,8 @@
 #pragma once
 
+#define Py_BUILD_CORE
```

Comment:
Why is this needed?

Reply:
This was conflicting when defining some GC/weakref structs. But these are not needed anymore!